lib/tasks/npm-task.js

```diff
@@ -17,6 +19,8 @@ class NpmTask extends Task {
   constructor(options) {
     super(options);
 
+    this.useYarn = ('useYarn' in options) ? options.useYarn : undefined;
```

Comment:
This seems odd to me since `options.useYarn === ('useYarn' in options) ? options.useYarn : undefined` anyways.